frontend/src/routes/(authenticated)/org/[org_id]/OrgTabs.svelte

```diff
@@ -14,29 +14,26 @@
 
 <script lang="ts">
   import t, { number } from '$lib/i18n';
-  import { createEventDispatcher } from 'svelte';
-
-  const dispatch = createEventDispatcher<{
-    clickTab: OrgTabId
-  }>();
 
   interface Props {
     hideSettingsTab?: boolean;
     activeTab?: OrgTabId;
     projectCount: number;
     memberCount: number;
+    onClickTab?: (tabId: OrgTabId) => boolean | undefined;
```

Comment:
Looks like we could remove this. I guess it mimics js/dom event handling, but it's the only place we return a boolean from a click handler and it's not used.
```suggestion
```

Reply:
Yeah, we haven't had any need to prevent org tab navigation. I want to keep the `onClickTab` function prop to parallel AdminTabs, but there's no need for it to return a boolean. And if nobody uses it, then we pass `undefined` to the event handler and Svelte is smart enough not to hook up an event handler, so there's pretty much no runtime cost to keeping it.

So I'm going to change the return type to `void` but keep the prop.

Reply:
... Actually, it doesn't parallel AdminTabs, In AdminTabs, the calling component is responsible for switching tabs; AdminTabs doesn't switch tabs when the tab is clicked on, and that's the purpose of `onClickTab`. OrgTabs handles its own tab-switching, so we *shouldn't* have the handler after all so that we don't confuse ourselves.